proto/regen/ecocredit/v1/tx.proto

```diff
@@ -133,30 +145,43 @@ message MsgCreateBatch {
   // quantified and verified.
   google.protobuf.Timestamp end_date = 6 [ (gogoproto.stdtime) = true ];
 
-  // BatchIssuance represents the issuance of some credits in a batch to a
-  // single recipient.
-  message BatchIssuance {
+  // If open is true we will enable future minting.
+  // Otherwise we will seal the batch and disable the future minting.
+  bool open = 7;
 
-    // recipient is the account of the recipient.
-    string recipient = 1;
+  // A reference to a transaction or an event referencing the transaction
+  // which caused the transfer from other chain or registry.
+  // If not set (null), it means that the batch originate in Regen Ledger
+  // (doesn't come from external registry).
+  OriginTx origin_tx = 8;
 
-    // tradable_amount is the number of credits in this issuance that can be
-    // traded by this recipient. Decimal values are acceptable.
-    string tradable_amount = 2;
+  // reference note for accounting, will be passed to an event
+  string note = 9;
+}
 
-    // retired_amount is the number of credits in this issuance that are
-    // effectively retired by the issuer on receipt. Decimal values are
-    // acceptable.
-    string retired_amount = 3;
+// BatchIssuance represents the issuance of some credits in a batch to a
+// single recipient.
+message BatchIssuance {
```

Comment:
This will also break tests:

```
core/msg_create_batch_test.go:32:18: undefined: MsgCreateBatch_BatchIssuance
```

Can you update any references of `MsgCreateBatch_BatchIssuance` to `BatchIssuance` before merging?